release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2956,3 +2956,22 @@ link:https://access.redhat.com/solutions/6962052[{product-title} 4.10.17 contain
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-z"]
+=== RHBA-2022:xxxx - {product-title} 4.10.z bug fix update
+
+Issued: yyyy-mm-dd
+
+{product-title} release 4.10.z is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.z --pullspecs
```

Comment:
I think this is a fine way to provide the data, and I'm glad that support suggested it. I have two requests, then it LGTM:
* Swap out (or explain) the`.z` part of this command 
* Confirm that you don't need to update `oc` before you run the command